### PR TITLE
Move command line parsing to be a workbench contribution

### PR DESCRIPTION
**Pull request by anthonydresser**
Move command line parsing to be a workbench contribution from a service.

The "service" won't be injectable anymore, but it wasn't being injected anyways. This makes more sense for this feature.

I tested with the --server argument and it worked.

### CODE REVIEW

**Comment by Charles-Gagnon** [comment]
> 	function getCommandLineService(connectionManagementService: IConnectionManagementService,

getCommandLineContribution (and update var names too here and below)

---
Refers to: src/sql/workbench/parts/commandLine/test/common/commandLine.test.ts:101 in edfc1c9. [](commit_id = edfc1c98b264cdcf8620541311f2870e4162460d, deletion_comment = False)

File: src/vs/workbench/workbench.main.ts [hunk 1]
```diff
@@ -525,3 +525,5 @@ import 'sql/workbench/parts/dashboard/browser/containers/dashboardContainer.cont
 import 'sql/workbench/parts/dashboard/browser/containers/dashboardNavSection.contribution';
 import 'sql/workbench/parts/dashboard/browser/containers/dashboardModelViewContainer.contribution';
 import 'sql/workbench/parts/dashboard/browser/core/dashboardTab.contribution';
+
+import 'sql/workbench/parts/commandLine/electron-main/commandLine.contribution';
```
Comment (Charles-Gagnon) [inline src/vs/workbench/workbench.main.ts]: >electron-main [](start = 40, length = 13)

So side note - but do we have a list anywhere of what things belong under electron-main/browser/electron-browser/node/etc? If not that would be awesome to have...I know I at least don't recall off hand all the different ones and what belongs under each folder type.

Comment (anthonydresser) [reply src/vs/workbench/workbench.main.ts]: https://github.com/microsoft/vscode/wiki/Code-Organization

**Comment by anthonydresser** [comment]
Tested with SSMS and it work for new notebook and new query

**Pull request merged**